frontend/public/components/network-policy.jsx

```diff
@@ -173,9 +173,10 @@ const IngressRow = ({ ingress, namespace, podSelector }) => {
   );
 };
 
-const Details_ = ({ obj: np }) => {
-  const networkPolicyDocs =
-    'https://kubernetes.io/docs/concepts/services-networking/network-policies/';
+const Details_ = ({ obj: np, flags }) => {
+  const networkPolicyDocs = flags[FLAGS.OPENSHIFT]
+    ? 'https://docs.openshift.com/container-platform/4.2/networking/configuring-networkpolicy.html'
```

Comment:
This needs to be based on the doc root defined in the console config. See https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx

It would be good to centralize our doc links in one place like we had previously: https://github.com/openshift/console/commit/2dae49a86c0d3a34c6ed969a575be3c54d47195a#diff-12887fc2371d38a701d3799cb2766141

Reply:
@spadgett - I want to do something like this:
```
 const networkPolicyDocs = flags[FLAGS.OPENSHIFT]
    ? `${openshiftHelpBase}networking/configuring-networkpolicy.html'`
    : 'https://kubernetes.io/docs/concepts/services-networking/network-policies/';
```
But that does not to work for okd (the OPENSHIFT flag is still true for okd) since okd defaults back to https://docs.okd.io/latest/.
Should I use the branding flag to decide when to use the ocp docs versus defaulting back to the k8s docs? 
```
const networkPolicyDocs = flags[FLAGS.OPENSHIFT] && window.SERVER_FLAGS.branding === 'ocp'
    ? `${openshiftHelpBase}networking/configuring-networkpolicy.html'`
    : 'https://kubernetes.io/docs/concepts/services-networking/network-policies/';
 ```

Reply:
There isn't an official OKD 4.x release yet. When we have one, the docs should updated. I'd anticipate that the same link works for OKD, so wouldn't put in special logic.